Makefile

```diff
@@ -11,7 +11,7 @@ ifneq ($(DKMS_BUILD),)
 # DKMS
 KERN_DIR := /lib/modules/$(KERNELRELEASE)/build
 
-ccflags-y := -Wno-error -isystem include/uapi/drm $(CFLAGS) $(EL8FLAG) $(EL9FLAG) $(RPIFLAG)
+ccflags-y := -Werror -isystem include/uapi/drm $(CFLAGS) $(EL8FLAG) $(EL9FLAG) $(RPIFLAG)
```

Comment:
maybe just remove the flag to use kernel's default? drm-loopback should build 4.14-6.12, it will not be trivial to have no warnings at all, constantly. using Werror from kernel allows people who care to have Werror, and everyone else won't.